### PR TITLE
Restore Apply for teacher training

### PR DESCRIPTION
**Pull request by duncanjbrown**
We have a process now, so put Apply back on the board

### CODE REVIEW

**Review by cwrw — APPROVED** [review]
looks, only wondering why we touched more files here https://github.com/DFE-Digital/teacher-services-upptime/pull/5/files than in this pr? (which are autogenerated etc.)

**Comment by saliceti** [comment]
> looks, only wondering why we touched more files here https://github.com/DFE-Digital/teacher-services-upptime/pull/5/files than in this pr? (which are autogenerated etc.)

Yes lots of files are autogenerated by upptime. It's a bit confusing!

File: .upptimerc.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ sites:
     url: https://www.find-postgraduate-teacher-training.service.gov.uk/
   - name: Publish teacher training courses
     url: https://www.publish-teacher-training-courses.service.gov.uk/
+  - name: Apply for teacher training
+    url: https://www.apply-for-teacher-training.service.gov.uk/
```
Comment (saliceti) [inline .upptimerc.yml:19]: Is home page ok or shall we use a more involved healthcheck? (as discussed for find and publish)

Comment (duncanjbrown) [reply .upptimerc.yml:19]: Oops - just merged on phone and saw this! Yes, we should probably use the proper healthcheck endpoint. I'll raise

**Pull request merged**